TA_dataset/bin/dataset_common.py

```diff
@@ -212,7 +279,36 @@ def get_acct_info(self, logger, account=None):
     return acct_dict
 
 
-def get_token(self, account, rw, logger):
+def get_tenant_value(conf, logger):
+    tenant_value = conf.tenant
+    tenant_value = tenant_value.strip()
+    logger.info("The provided tenant value in config is {}".format(tenant_value))
+    if tenant_value.lower() == "false" or tenant_value.lower() == "0":
+        return False
+    return True
+
+
+def get_account_ids(conf, logger):
+    account_ids_array = []
+    tenant = False if conf.tenant == "0" else True
+    if tenant:
+        logger.debug("Account configured on global level")
+        return account_ids_array
+    if hasattr(conf, "account_ids"):
+        account_ids_conf = conf.account_ids
+        account_ids_conf = account_ids_conf.strip()
+        if account_ids_conf:
+            account_ids_array = account_ids_conf.split(",")
```

Comment:
Do we validate trailing comma? eg. `accountId1,accountId2,` ideally we ignore empty part after trailing comma or rise an validation exception

Reply:
already handled

Reply:
where? `.strip()` handles whitespaces not comma

Reply:
Splunk provides the config validations it uses the regex as the validators. For account Ids used this regex `^\\d+(,\\d+)*$` as a validator. this will check the comma trialing exception, if any trialing comma occurs it wont allow the user to save the configuration.